README.md

```diff
@@ -180,6 +180,7 @@ Options:
   --bucket-name TEXT  Bucket name where Kedro Viz will be hosted  [required]
   --include-hooks     A flag to include all registered hooks in your Kedro
                       Project
+  --include-preview   Enable/disable preview for all the datasets.
```

Comment:
Looking at this, I am thinking if we should make this flag a plural form (--include-previews ), similar to --include-hooks as this flag deals with all the dataset previews. Since we haven't shipped this yet, it would be good to address this now. What do you all think ? @jitu5 @stephkaiser @Huongg

Reply:
@ravi-kumar-pilla I think it make sense to make a plural form `--include-previews` as this flag deals with all the dataset previews.

Reply:
should we make the change in - https://github.com/kedro-org/kedro-viz/pull/1895 ? @jitu5 once the team approves this change ?

Reply:
We can keep in https://github.com/kedro-org/kedro-viz/pull/1895

Reply:
@ravi-kumar-pilla I just had a quick chat with @stephkaiser and she is fine with a plural form `--include-previews` as flag.

Reply:
Lets update the flag in #1895 and also the docs. thank you @jitu5

Reply:
Let's hold on this. Sorry for the confusion. --include-preview seems to be for each dataset (since a dataset has only 1 preview, it seems more appropriate if we think that way) while --include-hooks was a project level thing.

Reply:
@ravi-kumar-pilla As I discussed with @stephkaiser we agreed on `--include-previews` as flag. All the relevant changes in code I did in https://github.com/kedro-org/kedro-viz/pull/1895 PR and doc changes in this PR.